projects/fishtornado_xr/FishTornado.h

```diff
@@ -138,7 +138,7 @@ class FishTornadoApp
         static constexpr size_t kCount             = 8;
 
         ppx::metrics::Manager      manager;
```

Comment:
Adding @mdagois for input on this.
I think things are getting a bit weird here. Application has it's own metrics Manager, and there is common flags to enable-metrics and specify filename. Could we reuse that instead of adding "--ft-enable-metrics" and a new manager here?
With the new code StartRun() no longer returns a run, so we have to go through the manager. Since the manager is private in the application class we can no longer add metrics to the default metrics system. Perhaps we could expose GetMetricsManager in the Application class and go through that?

Reply:
Again, subsequent PRs address this. I tried to break things up.